docs/integrations/airbyte.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Airbyte
+description: Use Airbyte with lakeFS to easily sync data between applications and S3 with lakeFS version control. 
+parent: Integrations
+nav_order: 57
+has_children: false
+---
+
+[Airbyte](https://airbyte.io//) is an open-source platform to sync data from applications, APIs & databases
+to warehouses, lakes and other destinations. Using Airbyte's connectors you can get your data pipelines running
+to consolidate many input sources.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc .pb-5 }
+   
+## Using lakeFS with Airbyte
+The integration between the two open-source projects brings resilience and manageability when using Airbyte
```

Comment:
Never like the "resilience and manageability" wording. It's too general. I would state specifically what it does